### ISSUE TITLE
multiple react versions ? 

### ISSUE THREAD

**Issue by tnrich** [cssinjs/theming#46]
Hey there,

I'm getting the following in my console: 

```
$ yarn list react
yarn list v1.0.1
├─ react@16.0.0
└─ theming@1.1.0
   └─ react@15.6.2
```

Any idea why theming might be keeping around an extra copy of react? Thanks!

**Comment by hburrows** [comment]
Yeah... `theming` is currently dependent on react 15.6.2.  Should be updated soon.  The master branch of `theming` fixes this but it's not been released to npm yet

**Comment by kof** [comment]
I am updating it right now.

**Comment by kof** [comment]
released v1.2.0
